docs/reference/connection.rst

```diff
@@ -248,11 +248,11 @@ instance-level configuration object.
 
   * - **Environment variable**
     - **CLI flag**
-  * - ``EDGEDB_DATABASE``
+  * - ``EDGEDB_SERVER_DATABASE``
     - ``--database/-d <name>``
-  * - ``EDGEDB_USER``
+  * - ``EDGEDB_SERVER_USER``
     - ``--user/-u <user>``
-  * - ``EDGEDB_PASSWORD``
+  * - ``EDGEDB_SERVER_PASSWORD``
```

Comment:
I believe these are actually different variables. I think the originals were correct in this context. The `SERVER` variables specify the default database, user, and password that will be created when the server is brought up, but in the context of client connection, the user is specifying what they want to connect to, which doesn't use the `SERVER` variables.

Hey, @fantix, can you check my work on that? Does it sound right to you, or are these changes correct?

Reply:
I can confirm that this is the case, @raddevon.

These changes are wrong.